docs/site/downloads/brs-copy-en_gb-to-en_001.md

```diff
@@ -24,3 +29,23 @@ Options:
     2.  Then do a diff with HEAD to make sure it matches expectations
     3.  Then check in en\_oo1.xml and CompareEn.java
 
+## Review guidelines for inclusion in en_001
+
+World English i.e. en_001 is intended to be as neutral as possible. Only data for items which differ from en need to be upleveled.
+
+This would mean making en_001 have the following characteristics:
+- Date formats
+    - For full, long, medium formats use European style day, month, year. Numeric dates are also day, month, year order where both day and month are zero padded.
+    - en_001 includes a comma following the day name while en_GB does not.
+- Time formats - Use 12 hour clock, as this is the predominant format for English speaking locales ( ref : https://en.wikipedia.org/wiki/Date_and_time_representation_by_country)
+- Spelling of unit names - (metre vs. meter) - Since the BIPM spelling is “metre”, “centimetre”, etc., The “en_001” locale should prefer “metre” over the American spelling “meter”.
+- Compact decimal number formats currently inherit from en until there is evidence that the en_GB abbreviations are the most common globally.
+- For units such as gallon that have both a “US” version and an “imperial” version: Both versions should have the qualifier (“US” or “imperial”), should not remove the “imperial” as in the GB versions.
+- Time zone names: en_001 should us “∅∅∅” to cancel the short metazone names inherited from en since US short metazone names are not relevant for most of the world.
```

Comment:
```suggestion
- Time zone names: en_001 should use “∅∅∅” to cancel the short metazone names inherited from en since US short metazone names are not relevant for most of the world.
```